app/app.go

```diff
@@ -817,6 +864,15 @@ func (app *App) SetStoreUpgradeHandlers() {
 		// configure store loader that checks if version == upgradeHeight and applies store upgrades
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
+
+	if upgradeInfo.Name == "2.0.0beta" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{acltypes.StoreKey},
+		}
+
+		// configure store loader that checks if version == upgradeHeight and applies store upgrades
+		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
+	}
```

Comment:
what's the plan for this? One idea: we leave atlantic-1 using 1.x.x upgrades and cherry-pick out 2.0.0beta PRs so we theroetically never hit this

Reply:
does that mean we don't want to upgrade atlantic-1 anymore after the rebase? because a lot of the rebased changes are dependent on like accesscontrol and such which is part of 2.0.0